### DISCUSSION TITLE
建议增长发送文本消息的功能

### DISCUSSION THREAD

**Discussion by kala-mansi** [localsend/localsend#683] [L-简体中文] [answered]
经常，需要在设备之间发送一些短的文本消息，也是复制文字到另一个设备。 
考虑到localsend已经有了发送文件的功能，
能不能再增加一个发文本消息的功能？

**Comment by evoke322** [comment]
附议，希望可以考虑这个功能

**Comment by Mapaler** [comment, marked as answer]
本来就有发送文本的功能啊
![localsend_app_2023-09-19_19-36-39](https://github.com/localsend/localsend/assets/6565860/f385b99d-a697-4c3b-95e8-b4112a939bbe)

**Reply by evoke322** [reply]
呃抱歉表达不太完整，准确说应该是希望可以看到发送历史，这样方便多次发送接受
